blueflood-http/src/integration-test/resources/blueflood.properties

```diff
@@ -0,0 +1,17 @@
+# No need for zookeeper for integration testing.
+# I believe this is because zk is only/primarily used to coordinator multiple rollup nodes so a single shard doesn't
+# get rolled up multiple times, but if zk is missing, the rollup just happens anyway.
+ZOOKEEPER_CLUSTER=NONE
+
+# Some tests based on HttpIntegrationTestBase use these values to ensure CORS is enabled and working on the endpoints
+# under test.
+CORS_ENABLED=true
+CORS_ALLOWED_ORIGINS=test.domain1.com, test.domain2.com, test.domain3.com
+CORS_ALLOWED_HEADERS=test.domain1.com, test.domain2.com, test.domain3.com
```

Comment:
Are the values same for HEADERS,METHODS and MAX_AGE config?

Reply:
Oh, oops, good catch. Somehow when I read it, it looked like they were all set to the same values.

Reply:
[Fixed](https://github.com/rackerlabs/blueflood/compare/18ab8f47aa1036f2cd5c7afedaee57719ddb673a..6fd203f11c8553e8a511760080be12096265e2aa). What's odd is the tests passed with this messed up. I guess they're not very thorough.